server/server.js

```diff
@@ -66,35 +68,78 @@ function massToRadius(mass){
     return Math.sqrt(mass / Math.PI) * 10;
 }
 
-function movePlayer(player, target) {
-    var dist = Math.sqrt(Math.pow(target.y, 2) + Math.pow(target.x, 2));
-    var deg = Math.atan2(target.y, target.x);
-
-    var slowDown = Math.log(player.mass);
+function movePlayer(playerObjects, center, target) {
 
-    var deltaY = player.speed * Math.sin(deg)/ slowDown;
-    var deltaX = player.speed * Math.cos(deg)/ slowDown;
+    var xMin = xMax = playerObjects[0].x;
+    var yMin = yMax = playerObjects[0].y;
 
-    if (dist < (50 + player.mass)) {
-        deltaY *= dist / (50 + player.mass);
-        deltaX *= dist / (50 + player.mass);
+    for(var i=1; i < playerObjects.length; ++i)
```

Comment:
playerObjects.forEach() to keep the same syntax for each loop ? (pun not intended)
If i'm boring, tell me now :p